src/thread.h

```diff
@@ -66,10 +68,11 @@ struct Thread : public std::thread {
   HistoryStats history;
   MovesStats counterMoves;
   Depth completedDepth;
+  std::atomic_bool resetCalls;
 };
 
 
-/// MainThread is a derived classes used to characterize the the main one
+/// MainThread is a derived class with a specific overload for the the main thread
```

Comment:
ok thanks for reporting.

On Sat, Nov 14, 2015 at 12:01 PM, hxim <notifications@github.com> wrote:

> In src/thread.h
> <https://github.com/official-stockfish/Stockfish/pull/500#discussion_r44857350>
> :
>
> >  };
> >
> >
> > -/// MainThread is a derived classes used to characterize the the main one
> > +/// MainThread is a derived class with a specific overload for the the main thread
>
> i think only one "the" should be here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/official-stockfish/Stockfish/pull/500/files#r44857350>
> .
>